### PR TITLE
Change layout & properties of unofficial tournaments

### PR DESCRIPTION
**Pull request by martijn9612**
This fixes the width on mobile causing the text to be wider than the page.

### CODE REVIEW

**Comment by skamoen** [comment]
Yeeaaaa no.

image

image

**Comment by martijn9612** [comment]
Whoops, I thought the media query could be inside a class, my bad. All should be ok now.

File: app/scripts/unofficial-tournaments.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   {
     "title": "Achtung",
     "subtitle": "1 v 1",
-    "headerTitle": "Achtung die Curve",
+    "headerTitle": "Achtung die Kurve",
```
Comment (bvanderdrift) [inline app/scripts/unofficial-tournaments.json]: Nice spot

**Comment by bvanderdrift** [comment]
LGTM

**Pull request merged**